Meadowlark-js/tests/http/local.33b.superclass.http

```diff
@@ -295,4 +295,72 @@ reference-validation: true
 
 ###
 GET http://localhost:3000/local/v3.3b/ed-fi/schools?schoolId=123
-{{auth_header}}
\ No newline at end of file
+{{auth_header}}
+
+###
+# These set of tests are to show that given a speific type of education organization (in this case
+# a school) Make sure to 
```

Comment:
Can you clarify this comment?

Reply:
whoops, that's what happens when you get distracted! Fixed